app/src/common/shared/com/igalia/wolvic/addons/adapters/AddonsListViewAdapter.kt

```diff
@@ -285,6 +285,12 @@ class AddonsManagerAdapter(
             itemsWithSections.addAll(recommendedAddons)
         }
 
+        // Add experimental section and addons if available
```

Comment:
Nit: code is self-explanatory so no need for this comment.

Reply:
I was hoping to be consistent with the remaining sections like a comment heading, eg. [here](https://github.com/Igalia/wolvic/blob/e050c25f77eb701e06c511241b035db1a754a8f7/app/src/common/shared/com/igalia/wolvic/addons/adapters/AddonsListViewAdapter.kt#L281), [here](https://github.com/Igalia/wolvic/blob/e050c25f77eb701e06c511241b035db1a754a8f7/app/src/common/shared/com/igalia/wolvic/addons/adapters/AddonsListViewAdapter.kt#L287) , but I can remove it if unnecessary